backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -181,14 +181,14 @@ object RuntimeAttributesValidation {
     */
   def toRuntimeAttributeDefinition(validation: RuntimeAttributesValidation[_]): RuntimeAttributeDefinition = {
     val name = validation.key
-    val unusedBoolean = false // ... at the time of this writing
+    val attributeRequired = false // ... at the time of this writing
```

Comment:
I still can't trace where this variable is read. We can discuss in person what this is and how it should be computed and remove the comment if it's no longer necessary.